sdk/KoreBuild/modules/frameworkbuild/3-compile.targets

```diff
@@ -0,0 +1,75 @@
+<Project>
+  <PropertyGroup>
+  	<CompileDependsOn>$(CompileDependsOn);CompileProjects;CompileRuntimeStore</CompileDependsOn>
+  </PropertyGroup>
+
+  <Target Name="CompileProjects" DependsOnTargets="PrepareProjects">
+    <KoreBuild
+      Projects="@(KoreBuildProject)"
+      Targets="Package"
+      BuildInParallel="true"
+      Properties="$(RepoBuildProps)" />
+
+    <PropertyGroup>
+      <!-- compilation done! Skip in future invocations -->
+      <RepoBuildProps>$(RepoBuildProps);NoBuild=true</RepoBuildProps>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <SymbolsPackageArtifactFile Include="%(KoreBuildProject.RootDir)artifacts\build\*.symbols.nupkg" />
+      <PackageArtifactFile Include="%(KoreBuildProject.RootDir)artifacts\build\*.nupkg" Exclude="@(SymbolsPackageArtifactFile)" />
+      <AdditionalArtifactFiles Include="%(KoreBuildProject.RootDir)artifacts\**\*" Exclude="@(PackageArtifactFile)" Repository="%(KoreBuildProject.Name)" />
+    </ItemGroup>
+
+    <!-- TODO do we still need this? -->
+    <Copy SourceFiles="@(PackageArtifactFile)" DestinationFolder="$(BuildDir)" />
+    <Copy SourceFiles="@(SymbolsPackageArtifactFile)" DestinationFolder="$(ArtifactsDir)symbols\" />
+
+    <Copy
+      SourceFiles="%(AdditionalArtifactFiles.Identity)"
+      DestinationFolder="$(ArtifactsDir)%(Repository)\%(RecursiveDir)%(FileName)%(Extension)" />
+
+    <!-- TODO implement task. Reads a nupkg file and creates and item with metadata about its id, version, and file location -->
+    <GetPackageIdentity Files="@(PackageArtifactFile)">
+
+      <Output TaskParameter="PackageIdentity" ItemName="PackageReference" />
+    </GetPackageIdentity>
+
+    <!-- TODO maybe we can make this more generic? e.g. just assign a Category metadata? -->
```

Comment:
This might not be a great idea since categories combinations might not scale with the number of columns. With the 2 columns right now you already have 6 possibilities {ship,noship,shipoob}X{include,exclude}. I like the idea of using a set of MSBuild items with metadata and then filter on those instead: e.g.
```
<ItemGroup>
  <RuntimeStore Include="@(Packages)" Condition="'%(OptimizedCache)' == 'include'" />
  <Shipping Include="@(Packages)" Condition="'%(Category)' == 'ship'" />
  <ImpossiblePackages Include="@(Packages)" Condition="'%(Category)' == 'noship' AND '%(OptimizedCache)' == 'include'" />
</ItemGroup>
```